src/main/java/de/tum/in/www1/artemis/service/QuizExerciseImportService.java

```diff
@@ -104,6 +104,12 @@ private void copyQuizQuestions(QuizExercise importedExercise, QuizExercise newEx
             }
             else if (quizQuestion instanceof DragAndDropQuestion dndQuestion) {
                 if (dndQuestion.getBackgroundFilePath() != null) {
+                    // Check whether dndQuestion.getBackgroundFilePath() is actually a background file path
+                    // (which is the case when its path starts with "/api/files/drag-and-drop/backgrounds/")
+                    FileService.sanitizeByCheckingIfPathContainsSubPathElseThrow(
+                            // Turn dndQuestion.getBackgroundFilePath() into a URI analog to
+                            // https://github.com/ls1intum/Artemis/pull/7038/files#diff-d41031bc9d88710f3ba653294756465029245a3d3a8d1af479b8a6498c254bd3
```

Comment:
That is now the system wide implementation and simply the way the FileService works now. It seems out of place.
```suggestion
```